docs/builders/index.mdx

```diff
@@ -36,8 +36,8 @@ Then, run [`packer init`](/docs/commands/init).
 packer {
   required_plugins {
     vmware = {
-      version = ">= 1.0.3"
-      source = "github.com/hashicorp/vmware"
+      version = ">= 1.0.7"
```

Comment:
I'd prefer an `x.y.x` example semver format so that it doesn't need to be updated with each release.

I used similar [here](https://github.com/hashicorp/terraform-provider-vsphere/blob/main/docs/INSTALL.md) in the Terraform provider docs.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.